generators/server/templates/src/main/java/package/util/JHipsterProperties.java.ejs

```diff
@@ -5,10 +5,10 @@ import io.micronaut.context.annotation.ConfigurationProperties;
 @ConfigurationProperties("jhipster")
 public class JHipsterProperties {
 
-    private final Cache cache;
-    private final Mail mail;
-    private final Logging logging;
-    private final Metrics metrics;
+    private Cache cache;
```

Comment:
When these where final the values could not be overwritten via properties files.